app/views/alchemy/admin/partials/_remote_search_form.html.erb

```diff
@@ -11,7 +11,7 @@
   <%= hidden_field_tag("content_id", @content.blank? ? "" : @content.id) %>
   <div class="search_field">
   <%= render_icon('search') %>
-    <%= text_field_tag :query,
+    <%= text_field_tag "q[name_or_image_file_name_or_upload_hash_or_image_file_uid_cont]",
```

Comment:
I took that from the search query generated on the library search. But I did not dig into it far enough to see how it was generated.
html on admin/pictures: ```<input id="search_input_field" placeholder="search" type="search" name="q[name_or_image_file_name_or_upload_hash_or_image_file_uid_cont]">```